jsdrupal/drupal-admin-ui
# Adds a Docker setup for this project

## Issue
Setup is hard!

## Testing instructions
- Checkout this branch and ensure it's free of any previous build files (e.g. a clean checkout or `git clean -f -d -X`)
- Follow the new instructions in the README
- Make sure you can run Nightwatch tests
- Profit!
